pkgs/applications/networking/filebrowser/default.nix

```
@@ -55,7 +55,7 @@ buildGoModule rec {
     description = "Filebrowser is a web application for managing files and directories";
     homepage = "https://filebrowser.org";
     license = licenses.asl20;
-    maintainers = with maintainers; [ nielsegberts ];
+    maintainers = with maintainers; [ nielsegberts oakenshield ];
```

Are you formatting this file?

```suggestion
    maintainers = with maintainers; [ nielsegberts oakenshield ];
```
```suggestion
    maintainers = with maintainers; [
      nielsegberts
      oakenshield
   ];
```

---

i did not format it before. did it after this suggestion.